common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -115,20 +173,26 @@ def create_content_groups(self):
 
     def link_problems_to_content_groups_and_publish(self):
         """
-        Updates 3 of the 4 existing problems to limit their visibility by content group.
+        Updates 5 of the 6 existing problems to limit their visibility by content group.
         Publishes the modified units.
         """
         container_page = self.go_to_unit_page()
+        enrollment_group = 'enrollment_track_group'
 
-        def set_visibility(problem_index, groups):
+        def set_visibility(problem_index, groups, group_partition='content_group'):
             problem = container_page.xblocks[problem_index]
             problem.edit_visibility()
             visibility_dialog = ComponentVisibilityEditorView(self.browser, problem.locator)
-            visibility_dialog.select_groups_in_partition_scheme(visibility_dialog.CONTENT_GROUP_PARTITION, groups)
+            partition_name = visibility_dialog.CONTENT_GROUP_PARTITION
```

Comment:
Personally, I find this easier to read, though perhaps it is too long. If so, I would go with an if/else.

````
partition_name = visibility_dialog.ENROLLMENT_TRACK_PARTITION if group_partition == enrollment_group else visibility_dialog.CONTENT_GROUP_PARTITION